src/utils/alerts.js

```diff
@@ -1,94 +1,83 @@
-import sweetAlert from 'sweetalert';
-import 'sweetalert/dist/sweetalert.css';
+import sweetAlert2 from 'sweetalert2';
 
 export function noMetaMaskAlert() {
-  sweetAlert({
+  sweetAlert2({
     title: "Warning",
-    text: "You don't have Metamask installed. Check ICO Wizard GitHub for <a href='https://github.com/oraclesorg/ico-wizard' target='blank'>the instruction</a>.",
-    html: true,
+    html: "You don't have Metamask installed. Check ICO Wizard GitHub for <a href='https://github.com/oraclesorg/ico-wizard' target='blank'>the instruction</a>.",
     type: "warning"
   });
 }
 
 export function noContractDataAlert() {
-  sweetAlert({
+  sweetAlert2({
     title: "Warning",
-    text: "The crowdsale data is empty. There is nothing to deploy. Please, start ICO Wizard from the beginning.",
-    html: true,
+    html: "The crowdsale data is empty. There is nothing to deploy. Please, start ICO Wizard from the beginning.",
     type: "warning"
   });
 }
 
 export function noContractAlert() {
-  sweetAlert({
+  sweetAlert2({
     title: "Warning",
-    text: "There is no contract at this address",
-    html: true,
+    html: "There is no contract at this address",
     type: "warning"
   });
 }
 
 export function invalidCrowdsaleAddrAlert() {
-  sweetAlert({
+  sweetAlert2({
     title: "Warning",
-    text: "Invalid crowdsale address is indicated in config and/or in query string.",
-    html: true,
+    html: "Invalid crowdsale address is indicated in config and/or in query string.",
     type: "warning"
   });
 }
 
 export function invalidNetworkIDAlert() {
-  sweetAlert({
+  sweetAlert2({
     title: "Warning",
-    text: "Invalid network ID is indicated in config and/or in query string.",
-    html: true,
+    html: "Invalid network ID is indicated in config and/or in query string.",
     type: "warning"
   });
 }
 
 export function successfulInvestmentAlert(tokensToInvest) {
-  sweetAlert({
+  sweetAlert2({
     title: "Success",
-    text: "Congrats! You've successfully bought " + tokensToInvest + " tokens!",
-    html: true,
+    html: "Congrats! You've successfully bought " + tokensToInvest + " tokens!",
     type: "success"
-  }, function() {
-    window.location.reload();
+  }).then(function(result) {
+    result.value && window.location.reload();
```

Comment:
Yes, the `result.value` check is necessary. In case of dismissal (by <kbd>Esc</kbd> or outside click) the promise callback will be called and the result will be `{dismiss: reaon}`

Here's the docs section about handling dismissals: https://github.com/limonte/sweetalert2#handling-dismissals

Reply:
I see, thanks for the explanation.